PickingNumbers.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const fs = require('fs');
+
+process.stdin.resume();
+process.stdin.setEncoding('utf-8');
+
+let inputString = '';
+let currentLine = 0;
+
+process.stdin.on('data', function(inputStdin) {
+    inputString += inputStdin;
+});
+
+process.stdin.on('end', function() {
+    inputString = inputString.split('\n');
+
+    main();
+});
+
+function readLine() {
+    return inputString[currentLine++];
+}
+
+/*
+ * Complete the 'pickingNumbers' function below.
+ *
+ * The function is expected to return an INTEGER.
+ * The function accepts INTEGER_ARRAY a as parameter.
+ */
+
+function pickingNumbers(a) {
+       // Write your code here
+]
+        var max = 0
+        a.forEach( elem => {
+        var subarray = a.filter( subElem => elem - subElem === 0 || elem - subElem === 1);
+        if( subarray.length > max) max = subarray.length;
```

Comment:
This goal of finding maximum value and assigning found value to variable `max` can be done by:
`max = Math.max(max, subarray.length)`